javascript-data-model/order-history.js

```diff
@@ -0,0 +1,68 @@
+var orderHistory = [
+  {
+    date: 'August 4, 2020',
+    total: 34.00,
+    recipient: 'JS Masher',
+    orderNumber: 114 - 3941689 - 8772232,
```

Comment:
I put an additional comment on this line, but it was after the review, so you may not have seen it. This should be a string, the dashes here are being interpreted as minus operators by JavaScript, so this is not being stored the way you expected.